invenio_circulation/api.py

```diff
@@ -69,22 +69,18 @@ def is_item_available(item_pid):
     return search.execute().hits.total == 0
 
 
-def is_item_can_request(item_pid):
-    """Return True if the given item can be requested, False otherwise."""
+def can_be_requested(loan):
+    """Return True if the given record can be requested, False otherwise."""
     config = current_app.config
-    cfg_item_can_request = \
-        config["CIRCULATION_POLICIES"]["request"]["can_request"]
-
-    return cfg_item_can_request(item_pid=item_pid)
-
-
-def is_document_can_request(document_pid):
-    """Return True if the given document can be requested, False otherwise."""
-    config = current_app.config
-    cfg_document_can_request = \
-        config["CIRCULATION_POLICIES"]["request"]["can_request"]
-
-    return cfg_document_can_request(document_pid=document_pid)
+    allow_request = config["CIRCULATION_POLICIES_ALLOW_REQUEST"]
```

Comment:
Question: did you add this extra check just to generally enable/disable?
I would prefer to "minimize" config whenever possible, I think this module has already a lot!

What about simply (just proposing!):
```
can_request = config["CIRCULATION_POLICIES"]['request'].get('can_request')
return can_request(loan) if can_request else True/False  # choose the default behaviour (I would propose False)
```
and we get rid of `CIRCULATION_POLICIES_ALLOW_REQUEST`.

Reply:
We have introduced this logic to make the `request` parameter on `CIRCULATION_POLICIES` configuration optional.
With the parameter `CIRCULATION_POLICIES_ALLOW_REQUEST`, it is possible to make a request without having to define the `CIRCULATION_POLICIES`.
Do you agree with this logic?